blatann/gatt/gatts.py

```diff
@@ -78,6 +79,7 @@ def __init__(self, read=True, write=False, notify=False, indicate=False, broadca
         self.user_description = user_description
         self.presentation = presentation_format
         self.sccd = sccd
+        self.cccd_security_level = cccd_security_level

```

Comment:
Could we change the name of this variable to be `cccd_write_security_level` since it only affects writes and not reads? Although a bit more verbose it is more clear in its intention

Reply:
Yes, sure. It makes the intention clearer